app/src/main/java/edu/cascadia/mobas/pollinatorpathway/ui/login/ProfileDao.java

```diff
@@ -13,19 +13,13 @@
 public interface ProfileDao {
 
     @Query("SELECT * FROM Profile WHERE userId = :userId")
-    public Profile getProfile(long userId);
+    public Profile getProfiles(long userId);
```

Comment:
To avoid confusion, use the plural form `getProfiles` only when you are returning a `List<Profile>` otherwise use `getProfile`